docs/development/setup_development.rst

```diff
@@ -8,12 +8,10 @@ Overview
 
 SecureDrop is a multi-machine design. To make development and testing
 easy, we provide a set of virtual environments, each tailored for a
-specific type of development task. We use Vagrant, VirtualBox, and
+specific type of development task. We use Vagrant, Molecule, and
```

Comment:
Do also want to include Libvirt in this list?

Reply:
I think that's a fine idea. Do you mind appending as part of review, @creviera ?